README.md

```diff
@@ -56,6 +51,19 @@
 
 * There are some suggested VSCode extensions in `.vscode/extensions.json` and additional docs at [docs/vscode.md](docs/vscode.md).
 
+## Linking and Unlinking with Force
+
+To _link_ your local reaction with your local force, run:
+
+        $ yarn link && yarn watch
+        $ cd ../force && yarn link @artsy/reaction && yarn start
```

Comment:
May want to mention here that waiting until `yarn watch` above prints the 'X files successfully compiled` babel message before starting this to avoid a race condition and ensure the module does exist when Force starts.

Reply:
updated!